corehq/apps/sms/static/sms/js/sms_usage_report.js

```diff
@@ -0,0 +1,15 @@
+hqDefine("sms/js/sms_usage_report",[
+    'jquery',
+    'knockout',
+    'hqwebapp/js/daterangepicker.config', //createDateRangePicker
+], function (
+    $
+) {
+    var labels = {
+        'last_7_days': 'Last 7 Days',
+        'last_month': 'Last Month',
+        'last_30_days': 'Last 30 Days',
+    };
+    var ele = $('#date_range_selector');
+    ele.createDateRangePicker(labels,' to ',ele.data('startDate'),ele.data('endDate'));
```

Comment:
Need to leave space after ",". I thought stickler caught this space syntax.

Reply:
Yeah. I kind of depend on my local git hook to identify these kinds of issues, I guess it doesn't work with JS 🤔